docs/setup-cluster/deploy-cluster/slurm/slurm-requirements.rst

```diff
@@ -202,6 +202,27 @@ interacts with Slurm, we recommend the following steps:
 Determined should function with your existing PBS configuration. To optimize how Determined
 interacts with PBS, we recommend the following steps:
 
+-  Enable PBS to store job history.
+
+   In order for the HPC launcher to detect successful job completion, PBS must be configured such
```

Comment:
Maybe something like:

For the HPC launcher to detect completion of a PBS job, the ``job_history_enable = True`` option must be set.  Otherwise, the HPC launcher will not be able to obtain information about the job after it completes.

Just my $0.02. Perhaps others may have a different opinion.